src/AdminConsole/Components/Pages/App/Settings/AuthenticationConfiguration/AuthenticationConfigurationForm.razor

```diff
@@ -0,0 +1,56 @@
+@using Fido2NetLib.Objects
+@inherits BaseRazorComponent
+
+@inject ILogger<AuthenticationConfigurationForm> Logger
+@inject IScopedPasswordlessClient Passwordless
+@inject ICurrentContext CurrentContext
+
+<EditForm Model="FormModel" OnValidSubmit="SubmitAsync" FormName="@FormName" class="space-y-6">
+    <DataAnnotationsValidator/>
+    <ValidationSummary/>
+    <div class="sm:w-full sm:max-w-md">
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Purpose">Purpose:</label>
+        <InputText @bind-Value="@FormModel!.Purpose" id="Purpose" disabled="@DisablePurpose" class="text-input"/>
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="UserVerification">User Verification:</label>
+        <InputSelect id="UserVerification" @bind-Value="@FormModel.UserVerificationRequirement" TValue="UserVerificationRequirement" class="form-select">
+            @foreach (var uv in Enum.GetValues(typeof(UserVerificationRequirement)))
+            {
+                <option value="@uv">@uv</option>
+            }
+        </InputSelect>
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Seconds">Seconds:</label>
+        <InputNumber id="Seconds" @bind-Value="@FormModel.Seconds" min="0" class="text-input"/>
+        <button class="btn-primary mt-3" type="submit">Submit</button>
+    </div>
+</EditForm>
+
+@code {
+    [Parameter] 
+    public EventCallback<AuthenticationConfigurationFormModel> OnSubmitFunction { get; set; }
+    
+    [Parameter]
+    public AuthenticationConfigurationFormModel? Model { get; set; }
+    
+    [Parameter]
+    public bool DisablePurpose { get; set; }
+    
+    
+    [SupplyParameterFromForm]
+    public AuthenticationConfigurationFormModel? FormModel { get; set; }
+
+    private const string FormName = "AuthenticationConfigurationForm";
+    
+    protected override void OnInitialized()
+    {
+        if (Model is not null && FormModel is null) FormModel = Model;

```

Comment:
FormModel will always be null here.

Reply:
On the initial load of the form, it will be null.